src/app/asset-page/asset-page.component.pug

```diff
@@ -18,73 +18,109 @@
               span.font-weight-normal &nbsp;of&nbsp;
               span#assetPageCount.font-weight-bold {{ totalAssetCount - restrictedAssetsCount }}
               i.icon.icon-direction.icon-right(title="Next in results", (click)="showNextAsset()", (keyup.enter)="showNextAsset()", [class.disabled]="assetNumber >= totalAssetCount - restrictedAssetsCount", [attr.tabindex]="assetNumber >= totalAssetCount - restrictedAssetsCount ? -1 : 1", aria-label="Next item, press enter to go to next result item. You can also use left and right arrow keys to navigate between result items.", role="button")
-    //- for accessibility so that page has title
-    h1.sr-only(*ngIf="isFullscreen") Full screen mode
-    //- Asset Viewer Row
-    .asset-col([ngClass]="[ isFullscreen ? 'fullscreen-container' : 'col-md-8', showAssetDrawer && isFullscreen ? 'drawer' : '' ]")
-      ng-container(*ngIf="userSessionFresh")
-        //- Artstor Viewer
-        artstor-viewer(
-            *ngFor="let assetId of (isFullscreen ? assetIds : [assetIds[0]]); let i = index",
-            [testEnv]="_auth.getEnv() === 'test'",
-            [assetId]="assetId",
-            [encrypted]="encryptedAccess",
-            [groupId]="assetGroupId",
-            [zoom]="indexZoomMap[i]",
-            (assetMetadata)="handleLoadedMetadata($event, i)",
-            [ngClass]="[ 'asset-viewer-' + assetIds.length ]",
-            (fullscreenChange)="updateFullscreenVar($event)",
-            [index]="i",
-            [assetCompareCount]="assets.length",
-            [assetNumber]="assetNumber",
-            [assetGroupCount]="totalAssetCount - restrictedAssetsCount",
-            [assets]="assets",
-            [isFullscreen]="isFullscreen",
-            [showCaption]="showAssetCaption",
-            [quizMode]="quizMode",
-            [prevAssetResults]="prevAssetResults",
-            (removeAsset)="toggleAsset($event, i)",
-            (nextPage)="showNextAsset()",
-            (prevPage)="showPrevAsset()",
-            (multiViewPageViaArrow)="multiViewPageViaArrow()",
-            (multiViewPageViaThumbnail)="multiViewPageViaThumbnail()",
-            (assetDrawer)="toggleAssetDrawer(!showAssetDrawer)",
-            [legacyFlag]="false",
-            [openLibraryFlag]="fromOpenLibrary"
+
+    .col-md-8#artstor-viewer
```

Comment:
Wrap all elements that are going to be shown in full screen mode in one div, so that I can only call request full screen on this div instead of the whole body. This is to prevent other interactive elements that are not showing in full screen mode from being accessible.